wlroots/ffi_build.py

```diff
@@ -45,6 +45,22 @@ def check_version():
         )
 
 
+def has_xwayland() -> bool:
+    """
+    Check for XWayland headers. If present, wlroots was built with XWayland support, so
+    pywlroots can be too.
+    """
+    try:
+        FFI().verify("#include <xcb/xcb.h>")
```

Comment:
I changed this to `wlr/xwayland.h` but this spits out lots of errors:
```
In file included from wlroots/__pycache__/_cffi__x1da17ed2x3dc425c6.c:265:
/home/mcol/git/wlroots/include/wlr/xwayland.h:6:2: error: #error "Add -DWLR_USE_UNSTABLE to enable unstable wlroots features"
    6 | #error "Add -DWLR_USE_UNSTABLE to enable unstable wlroots features"
      |  ^~~~~
In file included from /home/mcol/git/wlroots/include/wlr/xwayland.h:15,
                 from wlroots/__pycache__/_cffi__x1da17ed2x3dc425c6.c:265:
/home/mcol/git/wlroots/include/wlr/types/wlr_compositor.h:6:2: error: #error "Add -DWLR_USE_UNSTABLE to enable unstable wlroots features"
    6 | #error "Add -DWLR_USE_UNSTABLE to enable unstable wlroots features"
      |  ^~~~~
..
<repeated for a bunch of other files>
```
There doesn't seem to be anything in the cffi docs about passing in build flags, and I couldn't find an alternative way to see if a header can be found. Perhaps you know of another way?

Reply:
We pass that flag here, in set_source, you can pass `define_macros=[("WLR_USE_UNSTABLE", None)],`

Reply:
Hmm, in that case what would the first argument `module_name` be?

Reply:
Ah! I didn't realize this didn't have `set_source`, but it looks like you pass that to `.verify` instead. See https://cffi.readthedocs.io/en/latest/cdef.html?highlight=verify#ffi-verify-in-line-api-mode